WordPress/Sniffs/PHP/YodaConditionsSniff.php

```diff
@@ -46,6 +46,8 @@ public function process_token( $stackPtr ) {
 		$beginners   = PHP_CodeSniffer_Tokens::$booleanOperators;
 		$beginners[] = T_IF;
 		$beginners[] = T_ELSEIF;
+		$beginners[] = T_RETURN;
+		$beginners = array_merge( $beginners, PHP_CodeSniffer_Tokens::$assignmentTokens );
```

Comment:
It is a separate issue, but I wonder if it would make more sense to create this list only once, and save it in a property.